data-raw/pharmaversesdtm.R

```diff
@@ -0,0 +1,36 @@
+# Title: SDTM domain datasets provided by R package pharmaversesdtm
+# Description: This script is used to snatch those SDTM datasets provided by
+#              pharmaversesdtm. These include datasets that are therapeutic area
+#              (TA)-agnostic (DM, VS, EG, etc.) as well TA-specific ones
+#              (RS, TR, OE, etc.). We export them to inst/domain as RDS.
+#
```

Comment:
Is there a reason to copy these datasets to {sdtm.oak}? We can probably add `pharmaversesdtm` as a dependency and load it when needed. I highly doubt we will need all these datasets.

Reply:
The only reason was to have ownership over these data. As these data become used for testing and examples, an upstream change might suddenly break the building of `{sdtm.oak}`.